project/sites/victimspaymentsboard/themes/victimspaymentsboard_theme/src/scss/components.scss

```diff
@@ -14,6 +14,14 @@
   }
 }
 
+@media (min-width: 766px) {
```

Comment:
This will probably work but we would normally declare this:
`@include breakpoint($tab) {`
breakpoint is another way to say min-width.
The $tab variable comes from init/_variables.scss. It just means if our screensizes ever change we only need to update it in one place and it fixes it everywhere.

Reply:
That makes sense! Done, thanks Adam